Source/WebCore/page/text-extraction/TextExtraction.cpp

```diff
@@ -609,7 +610,7 @@ RenderedText extractRenderedText(Element& element)
         return true;
     }();
 
-    std::sort(allTokensAndOffsets.begin(), allTokensAndOffsets.end(), [&](auto& a, auto& b) {
+    std::ranges::sort(allTokensAndOffsets, [&](auto& a, auto& b) {
         return ascendingOrder ? a.offset < b.offset : a.offset > b.offset;
```

Comment:
I would probably pull the ascendingOrder check out of the lambda and do this as:

```c++
if (ascendingOrder)
    std::ranges::sort(allTokensAndOffsets, [&](auto& a, auto& b) { return a.offset < b.offset; });
else
    std::ranges::sort(allTokensAndOffsets, [&](auto& a, auto& b) { return a.offset > b.offset; });
```